website/docs/dev/building.md

```diff
@@ -21,16 +21,16 @@ git clone git@github.com:alibaba/fluss.git
 The simplest way of building Fluss is by running:
 
 ```bash
-mvn clean install -DskipTests
+./mvnw clean install -DskipTests
 ```
 
-This instructs [Maven](http://maven.apache.org) (`mvn`) to first remove all existing builds (`clean`) and then create a new Fluss binary (`install`).
+This instructs [Maven](http://maven.apache.org) (ran via the [Maven Wrapper](https://maven.apache.org/wrapper/) `mvnw`) to first remove all existing builds (`clean`) and then create a new Fluss binary (`install`).
```

Comment:
I prefer to use the native `mvn` command in documentation for users familiar with maven, and add a note here to instruct users to use `./mvnw` to replace `mvn` and explain maven wrapper ensures the correct Maven version (3.8.6).